Sources/DVR/Session.swift

```diff
@@ -13,7 +13,9 @@ open class Session: URLSession {
     public let backingSession: URLSession
     open var recordingEnabled = true
     open var recordMode : RecordingMode = .once
-
+    open var filter : Filter?
```

Comment:
It might make things syntactically cleaner to specify a default do-nothing filter. From the initializer, that's what it seems you'd get anyhow.

Reply:
Yup, I made that change